packages/models-library/src/models_library/api_schemas_resource_usage_tracker/pricing_plans.py

```diff
@@ -11,13 +11,14 @@
     PricingPlanId,
     PricingUnitCostId,
     PricingUnitId,
+    UnitExtraInfo,
 )
 
 
 class PricingUnitGet(BaseModel):
     pricing_unit_id: PricingUnitId
     unit_name: str
-    unit_extra_info: dict
+    unit_extra_info: UnitExtraInfo
```

Comment:
If I am not mistaken this model is used in the api-server. Maybe it would be an idea to at least run the unit-tests there to check that they don't break by this change.

Reply:
This one is not the one, there is a special one just for Api-server and I didn't dare to touch it, so it should be fine :+1: we will see in the unit tests in the CI.